docs/spec/index.md

```diff
@@ -139,3 +139,30 @@ Because some events may overlap in time, it may not be possible to choose a sing
 - Events may have gaps between the end time of one event and the start time of the next. e.g. if an operator's layovers aren't represented by an event.
 - `start_time` may equal `end_time` for an event that's a single point in time (such as a report time) without any duration.
 - Recommended sort order: `service_id`, `run_id`, `event_sequence`.
+
+### `vehicle_assignments.txt`
```

Comment:
Please add an entry to this file in the `Files` section of this document.

Reply:
I don't see an in-page TOC ?